databend-jdbc/src/main/java/com/databend/jdbc/DatabendNodes.java

```diff
@@ -0,0 +1,48 @@
+package com.databend.jdbc;
+
+import java.net.URI;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class DatabendNodes implements DatabendNodeManager {
```

Comment:
i feel this class can be removed, as it simply stores the backend uris, maybe it's ok to just store these fields inside DatabendClient.java?

Reply:
I think this class is required, there exists some state to keep more than a list of uri, extract those things into a class make implementation clean.